concepts.py

```diff
@@ -0,0 +1,17 @@
+# test function
+import numpy as np
+
+def fibonacci(n):
+    """ This definition returns the first n Fibonacci numbers as numpy array. E.g. calling fibonacci(21)
```

Comment:
The docstring of this function is not exactly following the pep 257 docstring conventions guide https://peps.python.org/pep-0257//
> Unless the entire docstring fits on a line, place the closing quotes on a line by themselves.

Reply:
This was just a random function I added here to test if I could get the package to work!